README.md

```diff
@@ -1,64 +1,55 @@
 
 # PHPTAL - Template Attribute Language for PHP
 
-Master: [![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
+[![Monthly Downloads](https://poser.pugx.org/phptal/phptal/d/monthly)](https://packagist.org/packages/phptal/phptal)
+[![License](https://poser.pugx.org/phptal/phptal/license)](LICENSE)
+[![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
```

Comment:
The link to Pornel's travis is broken.. I've set up a new one: https://travis-ci.org/phptal/PHPTAL
New code is:
 `[![Build Status](https://travis-ci.org/phptal/PHPTAL.svg?branch=master)](https://travis-ci.org/phptal/PHPTAL)`